generators/app/templates/test/basic.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const execa = require('execa');
+const assert = require('yeoman-assert');
+
+describe('Basic project capabilities', function() {
```

Comment:
Should this test be outside of the generated app project?
I don't see a benefit for having this within a generated project along with the `mocha`, `yeoman-assert` and `execa` dependencies.